pkgs/top-level/all-packages.nix

```diff
@@ -18391,6 +18391,8 @@ with pkgs;
 
   global = callPackage ../development/tools/misc/global { };
 
+  glualint = haskell.lib.compose.justStaticExecutables haskell.packages.ghc90.glualint;
```

Comment:
The static build of glualint is there for distributing glualint to non-NixOS Linux distributions. For NixOS I think it should be fine to use the regular dynamically linked version.

As for the GHC version, glualint currently builds against 9.4. GHC 9.0 should also work, but I think it might be best to use Nixpkgs' default GHC version. At this moment I believe that's 9.2.

Reply:
Haskell executables are always statically linked, regardless of `justStaticExecutables` or not — that function only gets rid of extra libraries that are installed (profiling, static, dynamic, you name it).